modules/text_generation.py

```diff
@@ -183,7 +182,7 @@ def generate_reply(question, state, eos_token=None, stopping_strings=[]):
     output = input_ids[0]
 
     if shared.args.verbose:
-        print(f'\n\n{decode(input_ids[0])}\n--------------------\n')
+        print('\n\n{decode(input_ids[0], state["skip_special_tokens"])}\n--------------------\n')
```

Comment:
The removal of `f` here breaks verbose output

Reply:
Oops, fixed it, thank you.